fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,39 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+Preregister=`opts_GetOpt1 "--preregister" $@`                        # what to use to preregister BOLDs before FSL BBR - epi_reg (default) or flirt
+
+# Defaults
+
+Preregister=${Preregister:-epi_reg}
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+
+# -- Preregister code
+
+if [ ! "${Preregister}" = 'epi_reg' ]; then
```

Comment:
As long as both registration methods are doing the same type of registration, having an additional option that sometimes works better doesn't sound like it needs to be restricted to only legacy data mode.

Reply:
I'm ok to have it open. Again @mharms and @glasserm need to chime in so that I don't go back and forth changing the code.

Reply:
I agree that there doesn't seem a need to restrict this to LegacyStyleData mode.

Reply:
Ok. Check taken out.